src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/PostcodeValidator.java

```diff
@@ -1,24 +0,0 @@
-package uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.helper;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
-public class PostcodeValidator {
-
-    public static List<String> validatePostcode(Map<String, String> fieldsMap, String postcodeKey) {
```

Comment:
Just to confirm, this one was not even used anymore it seems? :)

Reply:
Yeah, we were using the one already extracted to bsp-common-lib